CHANGELOG.md

```diff
@@ -17,6 +43,7 @@
 
 **Merged pull requests:**
 
+- release 1.1.1 [\#56](https://github.com/camptocamp/puppet-systemd/pull/56) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I don't know if gcg could ignore this one

Reply:
fixed